packages/agoric-cli/src/commands/inter.js

```diff
@@ -138,7 +138,7 @@ const coerceBid = (offerStatus, agoricNames, warn) => {
  *
  * @param {import('@agoric/smart-wallet/src/offers.js').OfferStatus &
  *         { offerArgs: BidSpec}} bid
- * @param {import('agoric/src/lib/format.js').AssetDescriptor[]} assets
```

Comment:
Did the reflexive import not work?

Reply:
didn't try it, was in a manual mechanical mode